test/blackbox-tests/test-cases/depend-on/expand-aliases-rerun.t

```diff
@@ -25,3 +25,5 @@ Now we set (expand_aliases_in_sandbox), and re-run the action.
   > (expand_aliases_in_sandbox)
   > EOF
   $ dune build @foo
+  .
+  ./alias-dep
```

Comment:
This test demonstrates the issue being fixed

Reply:
I don't follow. Could you make a separate PR where this test fails, so it's easy to see the problem and that this PR actually fixes it?

Reply:
This test demonstrates the issue https://github.com/ocaml/dune/pull/8991

Reply:
Thanks, could you rebase this PR to pick it up?

Reply:
I've rebased and added a sentence that explains the output. Let me know if it's clear now.

Reply:
That helps, thanks!